### PR TITLE
The Misfit of Demon King Academy S2 & fix D.Gray-man S2 & Slam Dunk

### PR DESCRIPTION
**Pull request by Arial-Z**
Add The Misfit of Demon King Academy S2 cour 1 and 2
Correct D.Gray-man S2 start episode following change on Space brother.
I'm not sur if i'm using the correct number as you put -50 for S2 of Space brother (S1 have 51 ep and S2 48), i'm a little lost.
D.Gray-man S1 have 51ep and S2 52ep so i set start at -52 (51 of the first season +1)

### CODE REVIEW

**Comment by Arial-Z** [comment]
Also add Slam Dunk

**Comment by Arial-Z** [comment]
Fixed start numbers follow discord guidelines and added BOFURI

File: series-tvdb.en.yaml [hunk 1]
```diff
@@ -1139,6 +1148,20 @@ entries:
       - season: 2
         anilist-id: 21733
 
+  - title: "Slam Dunk"
+    seasons:
+      - season: 1
+        anilist-id: 170
+      - season: 2
+        anilist-id: 170
+        start: -25
```
Comment (reconman) [inline series-tvdb.en.yaml:1157]: Forgot that we have a feature to combine multiple seasons into one automatically. See One Piece in this mapping file.

Omitting the `start` for every season should have the same result as the current mapping.

Can you test it?

Same is true for Space Brothers and D.Gray-Man.

**Review by reconman — CHANGES_REQUESTED** [review]

**Comment by Arial-Z** [comment]
going to test it on my custom mappings file with slam dunk as i'm currently watching it

**Review by reconman — APPROVED** [review]

**Pull request merged**